custom/ewsghana/handlers/help.py

```diff
@@ -0,0 +1,33 @@
+from corehq.apps.products.models import SQLProduct
+from custom.ewsghana.handlers import HELP_TEXT
+from custom.ilsgateway.tanzania.handlers.keyword import KeywordHandler
+
+
+class HelpHandler(KeywordHandler):
+
+    def help(self):
+        self.respond(HELP_TEXT)
+
+    def handle(self):
+        topic = self.args[0].lower()
+        if topic == 'stock':
+            self.respond("Please send your receipts in the format "
+                         "' <Commodity code> <stock on hand > . <quantity received>'")
+        elif topic == 'stop':
+            self.respond("Text 'stop' to stop receiving text message reminders.")
+        elif topic == 'start':
+            self.respond("Text 'start' to get text message reminders every week to submit your stock reports.")
+        elif 'code' in topic:
+            codes = [c.code for c in SQLProduct.by_domain(self.domain).order_by('code')]
```

Comment:
it would be more efficient to use a query that uses `values_list` here and pass it `flat=True` since you're only looking for the only value